src/components/filters/DateFilter.tsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { useEffect, useState } from 'react';
+import { Column } from '@tanstack/react-table';
+
+// components
+import CustomDatePicker from '../DatePicker';
+
+export type Option = {
+  value: number | string | number[];
+  label: string;
+};
+
+function DateFilter({
+  column,
```

Comment:
- [ ] why are you passing the entire column object if you only use one setFilterValue method from it?